src/components/Expander/Expander.less

```diff
@@ -67,4 +67,35 @@
 	&-content {
 		overflow: hidden;
 	}
+
+	// kind
+	&-with-emphasis {
+
+		.lucid-Expander-header {
+			background: @color-backgroundColor;
+			border: 1px solid @color-borderColorLight;
+			padding: 12px;
+			display: flex;
+
+			.lucid-Expander-text {
+				font-size: 15px;
```

Comment:
This is another font size that should be reflected in `variable.less` but currently the naming isn't very good for font sizes so go ahead and leave this. More of a note to myself.